src/components/RejectDialog.tsx

```diff
@@ -4,13 +4,21 @@ import { Formik, Form } from 'formik';
 import Field from './Field';
 import * as Yup from 'yup';
 import styles from '../styles/Dialog.module.scss';
+import { DocumentState } from '../domain/document-submission';
+import { DocumentSubmissionRequest } from '../gateways/internal-api';
 
 const schema = Yup.object().shape({
-  reason: Yup.string()
+  rejectionReason: Yup.string()
     .required('Please give a reason')
     .min(2, 'Reason needs to be at least three characters'),
 });
 
+const initialValues = {
+  state: DocumentState.REJECTED,
+  rejectionReason: '',
+  staffSelectedDocumentTypeId: '',
```

Comment:
`staffSelectedDocumentTypeId` is optional on `DocumentSubmissionRequest` so I don't think we need to provide a default value here

Reply:
Good point, addressed